dags/infra/s3_buckets.py

```diff
@@ -17,6 +17,14 @@
     "deafrica_sentinel_2", "deafrica-sentinel-2"
 )
 
+SENTINEL_2_SERVICES_BUCKET = Variable.get(
+    "deafrica_sentinel_2_services", "deafrica-services"
```

Comment:
this `deafrica-services` bucket is too misleading, can it be specified.

Reply:
@pindge I had a chat with @tebadi who also doesn't know which DAG or task fills this bucket up. I was informed by her, that's where we stored S2 Gomedian product.
I have created this variable, and all Sentinel 2 bucket variables, because they were hardcoded and infra wouldn't have any way to managing that.

Reply:
deafrica-services is a PDS bucket for Africa products and the name was agreed by the Africa team.  I'm not aware of any DAGs that write into this bucket, but Africa geomedian product was stored in this bucket.